plugins/woocommerce-admin/client/two-column-tasks/sectioned-task-list.scss

```diff
@@ -0,0 +1,84 @@
+.woocommerce-sectioned-task-list {
+	.components-panel {
+		width: 100%;
+		background: transparent;
+		border: 0;
+	}
+
+	.components-panel__body {
+		padding-bottom: 0;
```

Comment:
I'm seeing a small gap still below all of my sectioned task lists:
<img width="386" alt="Screen Shot 2022-04-12 at 3 01 24 PM" src="https://user-images.githubusercontent.com/10561050/163034861-31f08675-b3ab-4ee3-9116-507518af3aa1.png">

Reply:
Fixed as part of this commit: https://github.com/woocommerce/woocommerce/pull/32302/commits/0c91a56f1f6af19db974bea6a6cb1dc448d89162